### PR TITLE
Get rid of the wrap parameter

### PR DESCRIPTION
**Pull request by esoergel**
@dannyroberts
https://github.com/dimagi/commcare-hq/pull/7161/files?diff=unified#r32841283

### CODE REVIEW

File: corehq/apps/locations/tests/test_dbaccessors.py [hunk 1]
```diff
@@ -43,3 +43,9 @@ def test_get_user_ids_by_location(self):
     def test_get_one_user_at_location(self):
         user = get_one_user_at_location(self.meereen._id)
         self.assertIn(user._id, [self.tyrion._id, self.daenerys._id])
+
+    def test_get_user_docs_by_location(self):
+        users = get_user_docs_by_location(self.meereen._id)
+        import ipdb; ipdb.set_trace()
```
Comment (dimagimon) [inline corehq/apps/locations/tests/test_dbaccessors.py]: E702 multiple statements on one line (semicolon)

Comment (dannyroberts) [inline corehq/apps/locations/tests/test_dbaccessors.py]: @esoergel oops!

Comment (esoergel) [inline corehq/apps/locations/tests/test_dbaccessors.py]: :fearful:

File: corehq/apps/locations/tests/test_dbaccessors.py [hunk 2]
```diff
@@ -43,3 +43,8 @@ def test_get_user_ids_by_location(self):
     def test_get_one_user_at_location(self):
         user = get_one_user_at_location(self.meereen._id)
         self.assertIn(user._id, [self.tyrion._id, self.daenerys._id])
+
+    def test_get_user_docs_by_location(self):
+        users = get_user_docs_by_location(self.meereen._id)
+        self.assertItemsEqual([u['doc'] for u in users],
+                              [self.tyrion.to_json(), self.daenerys.to_json()])
```
Comment (dannyroberts) [inline corehq/apps/locations/tests/test_dbaccessors.py]: Nice work

**Pull request merged**